tests/schemas/api_v3/networkStatus.schema.js

```diff
@@ -38,15 +37,15 @@ const seedPeerSchema = {
 };
 
 const networkSchema = {
+	version: Joi.string().required(),
 	port: Joi.number().port().required(),
 	seedPeers: Joi.array().items(seedPeerSchema).required(),
 };
 
 const networkStatusSchema = {
 	version: Joi.string().pattern(regex.SEMVER).required(),
 	networkVersion: Joi.string().required(),
-	networkIdentifier: Joi.string().min(1).max(64).pattern(regex.HASH_SHA256)
-		.required(),
+	chainID: Joi.string().pattern(regex.CHAIN_ID).required(),
 	lastBlockID: Joi.string().min(1).max(64).pattern(regex.HASH_SHA256)
 		.required(),
```

Comment:
Also, please update the regex to be exactly 64 characters: `const HASH_SHA256 = /^\b([A-Fa-f0-9]){64}\b$/;`
```suggestion
	lastBlockID: Joi.string().pattern(regex.HASH_SHA256).required(),
```

Reply:
applied.